### DISCUSSION TITLE
spi1 on BD28 series

### DISCUSSION THREAD

**Discussion by PrefersWater** [SpenceKonde/DxCore#379] [Q&A]
hello sir/ma'am,  im trying to use MFRC522 RFID Reader on avrDB28s SPI1  but cant really make it work. what i done so far is just insert the line below before  SPI.begin(). The whole code is just the "Dumpinfo" example from MFRC522 library.

SPI.swap(SPI1_SWAP0);

Am I missing some additional lines or my approach is just not correct? BTW SPI0 works just fine.

**Comment by SpenceKonde** [comment]
I think that bug was fixed in 1.5.0 which is ready for release, but the release script which has worked flawlessly since i got it years ago is now giving errors that I don't understand or know how to fix.

**Reply by PrefersWater** [reply]
I will try it again once 1.5 come out. Thank you for your time :)
